modules/apps/transfer/types/forwarding.go

```diff
@@ -56,14 +47,30 @@ func (fi ForwardingPacketData) Validate() error {
 		return errorsmod.Wrap(ErrInvalidForwarding, "memo specified when forwarding packet data hops is empty")
 	}
 
-	for _, hop := range fi.Hops {
-		if err := host.PortIdentifierValidator(hop.PortId); err != nil {
-			return errorsmod.Wrapf(err, "invalid source port ID %s", hop.PortId)
-		}
-		if err := host.ChannelIdentifierValidator(hop.ChannelId); err != nil {
-			return errorsmod.Wrapf(err, "invalid source channel ID %s", hop.ChannelId)
+	return nil
+}
+
+func ValidateHops(hops []Hop) error {
```

Comment:
This will pass if len(hops) is zero. maybe even if hops is nil?

Is this expected behaviour?

Reply:
Maybe you could add this function to the `internal` package of transfer, so that we don't need to make it public.

Reply:
Made it private and removed the separate test. It is really being tested multiple other places, so I think it should be fine :)